stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1778,7 +1778,7 @@ function(add_swift_target_library name)
       if(SWIFTLIB_SHARED)
         list(APPEND swiftlib_swift_compile_flags_all -D_WINDLL)
         if(SWIFTLIB_IS_STDLIB_CORE)
-          list(APPEND swiftlib_swift_compile_flags_all -DswiftCore_EXPORTS)
```

Comment:
Please leave the `swiftCore_EXPORTS` macro in place.  That is a standard macro that is supposed to be defined by CMake, but it is not because Swift implemented a variant of CMake in CMake's scripting language.

Reply:
Hmm.  If it's standard, I'll switch back, but I have to say that this is a terrible, undescriptive macro name.  How does CMake define it, based on the current library target?  We seem to use it much more flexibly than that.

Reply:
And the use in LLVM.h is very unfortunate; please find more semantic ways to express the properties you actually want to conditionalize on.

Reply:
Yes, it is defined by the name of the library itself.